book/src/faq.md

```diff
@@ -64,3 +64,34 @@ As far as generating opaque blobs of bytes with the correct size and alignment,
 transitively contain STL things. We generally recommend marking `std::.*` as
 opaque, and then whitelisting only the specific things you need from the library
 you're binding to that is pulling in STL headers.
+
+### How to deal with bindgen generated padding fields?
+
+Depending the architecture, toolchain versions and source struct, it is
+possible that bindgen will generate padding fields named `__bindgen_padding_N`. 
+As these fields might be present when compiling for one architecture but not
+for an other, you should not initialize these fields manually when initializing
+the struct. Instead, use the `Default` trait. You can either enable this when
+constructing the `Builder` using the `derive_default` method, or you can
+implement this per struct using:
+
+```rust
```

Comment:
This looks great, but as the examples are compiled, CI is failing. You need to add `,ignore` here and below, I think, to fix it.

Reply:
Thanks for reviewing. I have added `,ignore` in the commit below.